core/i18n/resources/en.js

```diff
@@ -337,13 +337,17 @@ Do you want to cancel them?`,
   processingChainView: {
     dateExecuted: 'Date executed',
     status: 'Status',
-    deleteConfirm: 'Delete this processing chain?\n$t(common.cantUndoWarning)',
+    deleteConfirm: `Delete this processing chain?
```

Comment:
since the deleteConfirm messages are very similar, can we have a common.deleteConfirm message like `Delete this {{item}}? ...`